components/service/fretboard/README.md

```diff
@@ -12,6 +12,213 @@ Use gradle to download the library from JCenter:
 implementation "org.mozilla.components:fretboard:{latest-version}
 ```
 
+### Filters
+Fretboard allows you to specify the following filters:
+- Buckets: Every user is in one of 100 buckets (0-99). For every experiment you can set up a min and max value (0 <= min <= max <= 100). The bounds are [min, max).
+
+- appId (regex): The app ID (package name)
+- version (regex): The app version
+- country (regex): country, pulled from the default locale
+- lang (regex): language, pulled from the default locale
+- device (regex): Android device name
+- manufacturer (regex): Android device manufacturer
+- region: custom region, different from the one from the default locale (like a GeoIP, or something similar).
+For this to work you must provide a RegionProvider implementation when creating the Fretboard instance.
```

Comment:
Let's wrap class names like  RegionProvider in the markup for inline code so that they look like `this`.